content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -1,17 +1,29 @@
 # Use models
 
 Whether you need to download your models to use them, or you're looking to set
-up some automation in CI/CD to deploy them, Iterative Studio provides these
+up some automation in CI/CD to deploy them, DVC Studio provides these
 capabilities.
 
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using Studio [REST API], `dvc get`, or DVC [Python API].
+versions using the DVC Studio [REST API], DVC [Python API], use the
+`dvc artifacts get` command or directly download your model from DVC Studio.
 
-### Using Studio API
+Prerequisites:
 
-Using Studio API will require you to add your remote storage secrets to
+- The DVC Studio project you like to download your model from needs access to
+  your [remote storage credentials].
+- Access to your [DVC Studio Access token].
+
+[remote storage credentials]:
+  /doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials
+[DVC Studio Access token]:
+  /doc/studio/user-guide/account-management#studio-access-token
```

Comment:
```suggestion
- Model stored with DVC with S3 or Azure [remote].
- The DVC Studio project you like to download your model from needs access to
  your [remote storage credentials].
- Access to your [DVC Studio Access token].

Without these prerequisites, you can still [download a model artifact with  DVC], but
unlike using DVC Studio, the client where you download it will need DVC installed and
need access to both the Git repository and remote storage. 

[remote]: /doc/user-guide/data-management/remote-storage
[remote storage credentials]:
  /doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials
[DVC Studio Access token]:
  /doc/studio/user-guide/account-management#studio-access-token
[download a model artifact with DVC]: /doc/command-reference/artifacts/get
```

Reply:
I think it's better to describe this up front than get into it with individual download methods. Ultimately, you can download using Studio or directly with DVC, regardless of the interface you use.